### ISSUE TITLE
Update my personal information page accessibility issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16221]
## What Happened?
1. With Keyboard linked to Android: the "Find your nearest VA location" hyperlink is not being focused and can not be accessed 
2. With Voice Access enabled on Android, the "Find your nearest VA location" hyperlink is not able to be accessed by saying the attached number (the hyperlink has no attached number) or by saying "touch find your nearest VA location"
3. With VoiceOver andTalkBack enabled on iOS and Android, the back button a11yHint is not being read
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)

## Specs:
- Device: Android - LG LM-Q720 Stylo 5 with Voice Access and TalkBack enabled
- Device: iOS - iPhone XR with VoiceOver enabled
- Bluetooth Keyboard

## Steps to Reproduce

For 1:
1. With Keyboard linked to Android, navigate to the How do I update my personal information page 
2. Press tab or any of the directional keys to get the keyboard focus on the back button
3. Press tab or the down/right arrow keys. The page will be in focus
4. Press tab or the down/right arrow keys. The Home nav tab will be in focus (skipping the find your nearest VA location hyperlink)

For 2: 
1. With Voice Access enabled on Android, navigate to the How do I update my personal information page 
2. Press the blue Voice Access button to enable Voice Access on the page
3. Note that the "Find your nearest VA location" hyperlink is not able to be accessed by saying an attached number (the hyperlink has no attached number) or by saying "touch find your nearest VA location"

## Desired Behavior
For 1:
- With Keyboard linked to Android, the "Find your nearest VA location" hyperlink will be able to be placed in focus and selected with the tab or navigational keys

For 2:
- Using Voice Access on Android, the "Find your nearest VA location" hyperlink will be able to be selected by saying the attached number or by saying "touch find your nearest VA location"

## Acceptance Criteria
See desired behavior

## Bug Severity
- 3 - Medium

## Linked Story
- Story #13891

## Screen Shot (s) / Recordings (s) and Additional Information
1. [Video of the keyboard bug on Android](https://drive.google.com/file/d/1-o6OW3b-WD0REmykEACudDYVq9T5kAZ5/view?usp=sharing)
2. Note that there is no attached number to the hyperlink  🔽
![Update Personal Info Hyperlink.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/ddc809fa-4bd5-4c90-bd8c-d93038ec9acb)

**Comment by digitalken** [comment]
PASS 2/2/2021 - Able to access link now.
